MapboxDirections/MBRouteOptions.swift

```diff
@@ -412,6 +423,15 @@ open class RouteOptions: NSObject, NSSecureCoding, NSCopying{
             params.append(URLQueryItem(name: "voice_instructions", value: String(includesSpokenInstructions)))
             params.append(URLQueryItem(name: "voice_units", value: String(describing: distanceMeasurementSystem)))
         }
+        
+        if !roadClassesToAvoid.isEmpty {
+            let allRoadClasses = roadClassesToAvoid.description.components(separatedBy: ",")
+            let firstRoadClass = String(describing: allRoadClasses.first)
+            if allRoadClasses.count > 1 {
+                assert(false, "`roadClassesToAvoid` only accepts one `RoadClasses`. Although \(roadClassesToAvoid.description) was specified, only \(firstRoadClass) will be used.")
```

Comment:
No road classes will be used, because this assertion will crash the application.